src/sentry/static/sentry/app/components/avatar/baseAvatar.jsx

```diff
@@ -74,9 +84,11 @@ class BaseAvatar extends React.Component {
   };
 
   buildUploadUrl = () => {
-    let {uploadId} = this.props;
+    let {uploadPath, uploadId} = this.props;
 
-    return `/avatar/${uploadId}/?${qs.stringify({s: this.getRemoteImageSize()})}`;
+    return `/${uploadPath || 'avatar'}/${uploadId}/?${qs.stringify({
+      s: 120,
```

Comment:
I changed this to always fetch the largest avatar size, otherwise smaller avatars look like 💩